### ISSUE TITLE
Windows 10 Build 14332 (or possibly 14328): NVDA fails to notice selection event for Start search suggestions

### ISSUE THREAD

**Issue by josephsl** [nvaccess/nvda#5901]
Hi all,

Yet another Insider build regression (possibly related to UIA): NVDA fails to recognize selection event for Start search suggestion items.

STR:

1. Install Insider fast ring build 14332 (or 14328).
2. Open Start, then at "Cortana" prompt, type something.

Expected: NVDA announces search results.
Actual: NVDA does not announce results.

Setup:
* OS: Windows 10 Redstone build 14328, 14332 (observed in build 14332)
* NVDA version: various recent next snapshots (starting with 14297)
* Reproducible: Yes
* Observed with other screen readers: no (at least Narrator does read search suggestions).

Technical: NVDA still recognizes search items (when moving the navigator object to a search suggestion item, dev info still says search item, confirming the fact that search item recognition is still alive). Because Narrator announces them automatically, I conclude that NVDA is somehow not recognizing UIA selection event. This happens no matter which UIA support is in use (observed in 13300 and 13302).
Thanks.

**Comment by jcsteh** [comment]
Is this perhaps just another occurrence of #5470? For example, does 
restarting NVDA fix the issue for a short while?

CC @MichaelDCurran.

**Comment by michaelDCurran** [comment]
For me it seems completely broken. Restart does not fix. Yet it still 
works with Narrator.

The normal event dying bug (XAML comboboxes etc) still is flaky, and a 
restart fixes it. So I think this may not be related.

I shall investigate further.

**Comment by michaelDCurran** [comment]
An update: The main reason this is broken is actually because Microsoft Edge has changed its UIA FrameworkID from InternetExplorer to MicrosoftEdge. Allowing Edge NVDAObjects to be used for this frameworkID fixes suggestion selections, and also allows browse mode in Microsoft Edge again. However, other Edge changes now cause NVDA TextInfo code to go into endless loops, and or cause Edge to freeze. I will investigate further next week.

**Comment by josephsl** [comment]
Hi,
Fixed as of latest next snapshot 13323 and Redstone build 14342 (might have been due to a KB introduced on May 18th). Keeping it open until Edge ID ticket is closed. Thanks.

**Comment by josephsl** [comment]
Hi,

All fixed and included thanks to reworked Edge framework ID modifications. Thanks.
